core/templates/components/entity-creation-services/topic-creation.service.ts

```diff
@@ -45,9 +45,7 @@ angular.module('oppia').factory('TopicCreationService', [
               $scope.topicName = '';
               $scope.abbreviatedTopicName = '';
               $scope.isTopicNameValid = function() {
-                return (
-                  $scope.topicName !== '' &&
-                  $scope.topicName.length <= 20);
+                return $scope.topicName !== '';
```

Comment:
Shouldn't there still be a length limit check? If there's some other reason for omitting it, perhaps leave a comment to explain.

Reply:
Not required due to the maxlength attribute on input, will add a comment.

Reply:
Done